actions/login/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Container image that runs your code
```

Comment:
I am new to Github actions, and I have no idea where to plug in this file. Could we have some link or reference in the docs on how to spin the docker image with Github action?

Reply:
Right, as a developer if I use the action in the workflow there is no need for me to worry about the docker file. If we need to learn how to build GitHub actions then this can be used reference. Hope this makes sense.